cogs/archive.py

```diff
@@ -34,34 +33,66 @@ async def autocomplete_get_categories(ctx: TeXBotAutocompleteContext) -> Set[dis
         """
         Autocomplete callable that generates the set of available selectable categories.
 
-        The list of available selectable categories is unique to each member and is used in
-        any of the "archive" slash-command options that have a category input-type.
+        The list of categories only includes those that do not contain the word "archive".
         """
-        if not ctx.interaction.user:
+        try:
+            main_guild: discord.Guild = ctx.bot.main_guild
+        except BaseDoesNotExistError:
             return set()
 
-        try:
-            if not await ctx.bot.check_user_has_committee_role(ctx.interaction.user):
-                return set()
+        return {
+            discord.OptionChoice(name=category.name, value=str(category.id))
+            for category
+            in main_guild.categories
+            if "archive" not in category.name.lower()
+        }
```

Comment:
This circumvents permissions as it allows non-committee users to view channels they would not normally be able to see